src/modules/caen/CaenBase.h

```diff
@@ -25,8 +25,8 @@ class CaenBase : public Detector {
 
 protected:
   struct Counters Counters;
-  EV42Serializer *Serializer;
-  EV42Serializer *SerializerII;
+  EV44Serializer *Serializer;
+  EV44Serializer *SerializerII;
```

Comment:
Do we need SerializerII?

Reply:
Probably not, it is implementing a 'debug' format requested by Davide. I'll leave it for now and make a ticket.